File: user-portal/src/app/Models/account.ts
export class Account {
    customerId:number=0;
    accountNo:string="";
    ifsC_Code:string="";
    accountType:string="";
    accountBalance:number=0;
    branch:string="";
    
  static lastname: any;
​
    constructor(customerid:number, accountnum:string, ifsc_code:string, branch:string, accounttype:string, accountbal:number){
        this.customerId=customerid;
        this.accountNo=accountnum;
        this.ifsC_Code=ifsc_code;
        this.branch=branch;
        this.accountType=accounttype;
        this.accountBalance=accountbal;
    }
}

File: Backend/OnlineNetBankingWebAPI/Models/Account.cs
using System;
using System.Collections.Generic;

namespace OnlineNetBankingWebAPI.Models;

public partial class Account
{
    public int? CustomerId { get; set; }

    public string AccountNo { get; set; } = null!;

    public int? IfscCode { get; set; }

    public string? AccountType { get; set; }

    public double? AccountBalance { get; set; }

    public virtual UserDetail? Customer { get; set; }
}
